<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Tres En Raya</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/SuperTresEnRaya.png</normaloff>:/icons/SuperTresEnRaya.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJuego">
    <property name="title">
     <string>Juego</string>
    </property>
    <addaction name="actionCerrar"/>
    <addaction name="separator"/>
    <addaction name="actionResetear"/>
   </widget>
   <widget class="QMenu" name="menuInformaci_n">
    <property name="title">
     <string>Informaci√≥n</string>
    </property>
    <addaction name="actionNormas"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuJuego"/>
   <addaction name="menuInformaci_n"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionResetear">
   <property name="text">
    <string>Resetear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNormas">
   <property name="text">
    <string>Normas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre el juego</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
